### PR TITLE
Feature/nz suburb bug fix

### PR DESCRIPTION
**Pull request by katenorquay**
Issue #4
Suburb field wasn't being populated into any field.

### CODE REVIEW

File: js/addressfinder/src/providers/nz.js [hunk 1]
```diff
@@ -2,10 +2,19 @@ module.exports = {
     country: 'NZ',
     map(fullAddress, metaData) {
         let selected = new AddressFinder.NZSelectedAddress(fullAddress, metaData);
+        let street1Value, street2Value
```
Comment (OliverGreenaway) [inline js/addressfinder/src/providers/nz.js]: Probably best to keep this file consistent with semi-colons after each of these new lines.

**Review by OliverGreenaway — CHANGES_REQUESTED** [review]
Great fix, just one small thing before this one can go out.

File: js/addressfinder/src/providers/nz.js (hunk 1 above)
Comment (katenorquay) [reply js/addressfinder/src/providers/nz.js]: 👍

**Review by OliverGreenaway — APPROVED** [review]
Looks great :shipit:

**Pull request merged**